common/dynamicconfig/constants.go

```diff
@@ -1291,6 +1291,11 @@ second per poller by one physical queue manager`,
 		60*time.Second,
 		`Timeout for forwarded backlog task (requires new matcher)`,
 	)
+	MatchingSpreadRoutingBatchSize = NewGlobalIntSetting(
+		"matching.spreadRoutingBatchSize",
+		0,
+		`If non-zero, try to spread task queue partitions across matching nodes better, using the given batch size.`,
```

Comment:
It seems the assumption is the batch size will be set to a number that is comfortable lower than the number of matching pods, right? Should we mention that here?

Reply:
Also should we mention the risk of changing this number after a cluster starts getting traffic?